src/main/java/io/cdap/plugin/gcp/bigtable/common/TableOutputCustomFormat.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.gcp.bigtable.common;
+
+import org.apache.hadoop.hbase.mapreduce.TableOutputFormat;
+import org.apache.hadoop.mapreduce.JobContext;
+
+import java.io.IOException;
+
+/**
+ * Table output format class with null credentials fix fon non-default service accounts
```

Comment:
Please add more details about how this class is used

Reply:
Also, why is this only needed in the table ouput format but not the input format?

Reply:
When using non-default credentials we faced null pointer exception issue  - this would happen only during sink operations. While researching the cause of the problem we found out that the reason behind it was ``checkOutputSpecs`` method which calls ``createConnection``, however at this stage the configuration is not set yet. In order to solve it we had to trigger ``setConf`` call before ``checkOutputSpecs`` call.

Reply:
I see, thanks for explaining!